tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -111,17 +111,22 @@ def test_no_configs(self):
 
     def test_file_not_found_error(self):
         """File matches code mapping but it cannot be found in the source repository."""
-        response = self.get_success_response(
-            self.organization.slug, self.project.slug, qs_params={"file": self.filepath}
-        )
-        assert response.data["config"] == self.expected_configurations(self.code_mapping1)
-        assert not response.data["sourceUrl"]
-        assert response.data["error"] == "file_not_found"
-        assert response.data["integrations"] == [serialized_integration(self.integration)]
-        assert (
-            response.data["attemptedUrl"]
-            == f"https://example.com/{self.repo.name}/blob/master/src/sentry/src/sentry/utils/safe.py"
-        )
+        with mock.patch.object(
+            ExampleIntegration,
+            "get_stacktrace_link",
+            return_value=None,
```

Comment:
Change the `test_file_not_found_error` function to mock `get_stacktrace_link` return value to be none, since we don't want it to be found

Reply:
Let's put this test the way it was. I will checkout out your PR once you make the changes.